src/main/java/com/badlogic/gdx/math/Vector4.java

```diff
@@ -0,0 +1,363 @@
+package com.badlogic.gdx.math;
+
+import java.io.Serializable;
+
+public class Vector4 implements Serializable, Vector<Vector4>{
```

Comment:
What's the rationale behind this class in this package?

Reply:
originally I was thinking about replacing the implementation with lbgdx. The math implementation has a lot of shortcoming. I think it's probably better to just close this.